sailon/random_novelty_detector.py

```diff
@@ -10,10 +8,55 @@
 
 from smqtk_core import Configurable, Pluggable
 
+from .errors import RoundError
+from .novelty_interface import ONDAdapter
+
+
 logger = logging.getLogger(__name__)
 
 
-class RandomNoveltyDetector(Configurable, Pluggable):
+class RandomNoveltyDetectorAdapter(ONDAdapter):
```

Comment:
Should we just rename this to RNDAdapter?

Reply:
`RandomNoveltyDetectorAdapter` is a long name, but I think long/descriptive names are better for clarity than abbreviations (within reason). In this case, I'd also be worried of that naming scheme getting confusing with, for example, the "nd_rd" part of the filenames of some of the GAE modules, e.g. `gae_nd_rd_adapter.py`